include/openmc/openmp_interface.h

```diff
@@ -7,6 +7,27 @@
 
 namespace openmc {
 
+//==============================================================================
+//! Accessor functions related to number of threads and thread number
+//==============================================================================
+inline int openmc_n_threads()
```

Comment:
These are already in the `openmc` namespace; we don't need the `openmc_` prefix on the function name since they are not being exposed through the C API.

Reply:
True true. Their origin code seems a little ambiguous without the prefix though. Something like the following doesn't indicate to me that the determination of the number of threads is coming from OpenMC:

```cpp
for (int t = 0; t < n_threads(); t++) {
...
}
```

Reply:
You could say the same about any free function within OpenMC. One could write `openmc::n_threads()`, but generally it's safe to assume that if we are calling a non-namespaced function, it is coming from the `openmc` namespace.

Reply:
True, though they do typically have more domain-specific names that make it clear the functionality is part of OpenMC. At the same time, there are other general-purpose functions like `lower_bound_search` and many in the HDF5 interface that fall into this category. I think I was viewing this from a multi-application level in part. 

I'll adjust accordingly. Thanks!